other/lfu_cache.py

```diff
@@ -0,0 +1,188 @@
+from typing import Callable, Optional
+
+
+class DoubleLinkedListNode:
+    '''
```

Comment:
Please change `'''` to `"""` in docstrings

Reply:
autoblack will do this automatically so let's not slow the review process on this.